tests/examples/authors_combined_terms.example

```diff
@@ -0,0 +1,84 @@
+Test cases for author search tuning, updated 2018-04-30 EAW
+
+NOTE: It may seem that there are a lot of duplicate example cases, but it has been
+found during testing that not every case performs in a similar fashion. Some already
+work well under current rules, and others have trouble for sometimes mysterious reasons.
+It may be worth keeping a library of search cases that may have had trouble in the
+past to prevent behavior regression.
```

Comment:
Great idea! I wrote that note about regressions because I've been using this document to check our work against backsliding during this sprint. Might be a good idea to bring the list up into a place where it's easier to access and labeled more precisely as a manual regression test -- and add cases to it as we find and squash new bugs.